.github/config.yml

```diff
@@ -0,0 +1,30 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+# Heavily inspired by https://github.com/electron/electron/blob/master/.github/config.yml
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: |
+  👋 Thanks for opening your first issue here! If you're reporting a 🐞 bug, please make sure you include steps to reproduce it. We get a lot of issues on this repo, so please be patient and we will get back to you as soon as we can.
+  To help make it easier for us to investigate your issue, please follow the [contributing guidelines](https://github.com/videojs/video.js/blob/master/CONTRIBUTING.md).
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
+
+# Comment to be posted to on PRs from first time contributors in your repository
+newPRWelcomeComment: |
+  💖 Thanks for opening this pull request! 💖
+
+  Things that will help get your PR across the finish line:
+
+  - Run `npm run lint -- --errors` locally to catch formatting errors earlier.
```

Comment:
Actually since we have a git hook for this, it shouldn't be possible to push with linting errors

Reply:
you can turn it off with `--no-verify`

Reply:
yeah but then its pretty much your own fault haha